defaults/main.yml

```diff
@@ -42,7 +42,7 @@ satellite_volumes:
 ##### FIREWALL   #####
 ######################
 
-# List of service to add into the firewall via Firewalld
+# List of services to add into the firewall via Firewalld
```

Comment:
This might be one of the rare use cases where we should stash the required ports and services into vars/main.yml. 

Keep custom_firewall_X in defaults and the base ones over in vars/main.yml and should be good.